editor/svg-editor.js

```diff
@@ -2037,20 +2041,6 @@ TODOS
 							// Clicking the icon in flyout should set this set's icon
 							var func = function(event) {
 								var options = opts;
-								//find the currently selected tool if comes from keystroke
-								if (event.type === 'keydown') {
-									var flyoutIsSelected = $(options.parent + '_show').hasClass('tool_button_current');
-									var currentOperation = $(options.parent + '_show').attr('data-curopt');
-									$.each(holders[opts.parent], function(i, tool) {
-										if (tool.sel == currentOperation) {
-											if (!event.shiftKey || !flyoutIsSelected) {
-												options = tool;
-											} else {
-												options = holders[opts.parent][i+1] || holders[opts.parent][0];
-											}
-										}
-									});
-								}
```

Comment:
My commit message for that change said "Removed some code that would select the wrong tool when using keyboard shortcuts for flyout buttons." On the svg-edit demo page you might notice that the keyboard shortcut for the line tool [L] doesn't work properly:
https://svg-edit.github.io/svgedit/releases/svg-edit-2.8.1/svg-editor.html

I haven't gone back and tested it now, but i believe this code was part of the issue and didn't seem to serve a purpose. I haven't been able to find a keyboard shortcut that doesn't work as expected after this change.